### ISSUE TITLE
Add "period" keyboard shortcut

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1052]
Requested:

> One thing missing in pinafore is the ability to load new toots and jump to them using the new keyboard shortcuts. e. g. the equivalent of the period key on Twitter. If I'm a hundred or so toots down, and want to load new toots, I have to switch contexts to the tabs at the top using screen reader functions instead of just pressing a single key and continuing reading.
